day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,25 @@
+# Program that prints something for each number from 1 to 100
+# following several rules
+
+def fizz_buzz(n)
+  # Create array of integers 1 to n
+  num_array = (1..n).to_a
+  # Iterate through the elements of the array
+  num_array.each do |a|
+  # For any number that is a multiple of both 3 and 5, print 'FizzBuzz'
+    if (a % 3) == 0 && (a % 5) == 0
+      print "FizzBuzz, "
+# For any number that is a multiple of 3, print 'Fizz'
+    elsif (a % 3) == 0 && (a % 5) != 0
+      print "Fizz, "
```

Comment:
You don't need the second part of the conditionals here- if both applied, it'd be swooped up by your first condition! You can just run the second 2 conditions checking for the 0 value of a modulo, rather than needing to see if it != 0.